shscripts/renew-cert.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+certbot renew --quiet
```

Comment:
Hier könnte man vllt. direkt zu Anfang oder nach `certbot renew --quiet` einmal in den Ordner mit den Compose-Files wechseln. Dann muss man nicht vor jedem Ausführen Aufruf von `docker-compose` wechseln. Wobei das zweite `cd` eigentlich nicht mehr nötig sein sollte.

```sh
SCRIPT_DIR="$(dirname $0)"
cd $SCRIPT_DIR/..

certbot renew --quiet 
docker-compose stop api
sudo docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d api
```

Hab das Schnippsel nicht getestet.

Ansonsten kann man auch schauen, dass man die Commandos vllt. per `&&` verkettet. So werden nachfolgende Commandos nicht ausgeführt, sollte ein Commando fehlschlagen. Ist denke ich aber kein Muss.

Reply:
```
SCRIPT_DIR="$(dirname $0)" 
cd $SCRIPT_DIR/..
```
Das kannte ich noch gar nicht.
Habe das Script überarbeitet. Danke für's Reviewen :)